src/test/functional/lock-account-test.ts

```diff
@@ -0,0 +1,52 @@
+import {config as testConfig} from '../config';
+import {randomData} from './shared/random-data';
+import {createUserWithRoles} from './shared/testingSupportApi';
+
+Feature('Lock User Account');
+const DASHBOARD_USER_EMAIL = randomData.getRandomEmailAddress();
+const AFTER_RESET_PASSWORD = testConfig.PASSWORD + randomData.getRandomString(2);
+
+BeforeSuite(async () => {
+  await createUserWithRoles(DASHBOARD_USER_EMAIL, testConfig.PASSWORD, testConfig.USER_FIRSTNAME, [testConfig.RBAC.access]);
+});
+
+Scenario('I as a user when try to sign in should get locked after 5 login attempts with wrong password and should be able to login once I reset my password', async ({I}) => {
```

Comment:
This test is not testing anything on user dashboard. The sign in and reset password are using the existing web public behaviour. You want to test the user dashboard user details page the locked message appears when user is locked and disappears after he user reset password.